hw4/03-form.js

```diff
@@ -9,8 +9,22 @@ app.use(express.static('public'));
 app.use(express.urlencoded({ extended: false }));
 
 // POST request
-app.post('', (req, res) => {
+app.post('/submit', (req, res) => {
   // Add your code here
+  const output = (key, value) => {
+    const answer = value || 'n/a';
+    return `${key}: ${answer}<br />`;
+  };
+
+  res.status(200).set({ 'Content-Type': 'text/html' });
+
+  let body = '';
+  Object.entries(req.body).forEach(([key, value]) => {
```

Comment:
This is awesome! It makes everything simple and easy to execute for your form. I love how short your code it in this exercise!